website/versioned_docs/version-0.20/migration-guides/yew/from-0_19_0-to-0_20_0.mdx

```diff
@@ -50,6 +50,14 @@ The reducer function can see all previous changes at the time they are run.
 
 You need to enable feature `csr` to use `yew::Renderer`.
 
+For example, to use client side rendering to render a typical app component:
+
+```rust ,ignore
+yew::Renderer::<App>::new().render();
+```
+
+For more options, see [the docs](https://docs.rs/yew/latest/yew/struct.Renderer.html).
```

Comment:
Could you use a stable link here? Otherwise, looks good to me, it doesn't hurt to be more explicit.

```suggestion
For more options, see [the docs](https://docs.rs/yew/0.20.0/yew/struct.Renderer.html).
```